### ISSUE TITLE
Docs changes are not deploying

### ISSUE THREAD

**Issue by mrjones-plip** [medic/cht-docs#635]
## tl;dr

The GitHub Pages settings where docs are served from needs to be updated from `master` to `main`. We think [this is the setting URL](https://github.com/medic/medic.github.io/settings/pages) where that happens for the `medic.github.io` repo, but might be wrong.  Here's a screenshot (taken from another repo, obvs ;) 

![image](https://user-images.githubusercontent.com/8253488/153544572-0a2eccea-539d-402a-94ef-d35fe24c0c67.png)

## Background

PRs that merge to `main` branch trigger [GH Actions](https://github.com/medic/cht-docs/blob/main/.github/workflows/ci.yml) to, in turn,  push the changes to the [repo where the docs site is served from](https://github.com/medic/medic.github.io). The push from this repo to the other seems to be working, as we can see them in the `main` branch starting on [Jan 26, 2022](https://github.com/medic/medic.github.io/commits/main).  However, the changes in `main` are not being reflected on the public docs site. 

For example, on [this page](https://docs.communityhealthtoolkit.org/core/guides/android/development-setup/), it should show the timestamp as `2022-02-01` on the "Create documentation issue" link in the upper right. Instead it shows the old date `2021-11-08`:

```html
 <a href="https://github.com/medic/cht-docs/issues/new?title=Improve+%27Development%20Environment%27&labels=enhancement&body=Page:+/core/guides/android/development-setup/%0aVersion:+2021-11-08%2018:53:31%20-0300%20-0300%0a---" target=_blank><i class="fab fa-github fa-fw"></i>Create documentation issue</a>
```

This is because [this commit](https://github.com/medic/medic.github.io/commit/d33cb9cb97f0d5bb7bfa0d97b13b209e40bb97f8#diff-d839913cfd1210c072b70759b686206e2447271da475087330e1796c0d9c7266R121) isn't being shown.

cc @latin-panda

**Comment by latin-panda** [comment]
The setting has been updated and now it's taking the `main` branch for deployments (see slack tread in development channel)

How ever the default branch in https://github.com/medic/medic.github.io is still master. @jonathanvq I was wondering if you have access to the repo's settings and make `main` default branch? Thanks!

**Comment by mrjones-plip** [comment]
Made [a ticket for the default branch change](https://github.com/medic/cht-docs/issues/636) per the slack [thread](https://medic.slack.com/archives/C024KTGRW/p1644561681422069?thread_ts=1644558077.392279&cid=C024KTGRW):

> I think it'd be good to create a new issue to fix that (`master` -> `main`). There already is a main branch created on the repo, in case the switch was not made for a good reason.

closing this one - thanks all!
